SPECS/clamav/clamav.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
- "Signatures": {
-  "clamav-1.0.6.tar.gz": "7a9691525bd150f9419c337d25d8c28402ed35beea7aa19563aabcddc248de52",
-  "clamav-1.0.6-cargo.tar.gz": "20b6179033c32ad81023ccc4a4262730d10fa80cd7ae678b386f139305b1e31f"
- }
-}
\ No newline at end of file
+  "Signatures": {
+    "clamav-1.0.6-cargo.tar.gz": "20b6179033c32ad81023ccc4a4262730d10fa80cd7ae678b386f139305b1e31f",
```

Comment:
This cargo file likely needs to be upgraded as well, can you please re-generate it?

Reply:
Good catch (and the buddy build caught it too).

This had been generated and uploaded already for the [corresponding upgrade in 3.0](https://github.com/microsoft/azurelinux/pull/10778).